state/migration_import.go

```diff
@@ -2184,23 +2184,28 @@ func (i *importer) addAction(action description.Action) error {
 		Completed:  action.Completed(),
 		Status:     ActionStatus(action.Status()),
 	}
-	prefix := ensureActionMarker(action.Receiver())
-	notificationDoc := &actionNotificationDoc{
-		DocId:     i.st.docID(prefix + action.Id()),
-		ModelUUID: modelUUID,
-		Receiver:  action.Receiver(),
-		ActionID:  action.Id(),
-	}
+
 	ops := []txn.Op{{
 		C:      actionsC,
 		Id:     newDoc.DocId,
 		Insert: newDoc,
-	}, {
-		C:      actionNotificationsC,
-		Id:     notificationDoc.DocId,
-		Insert: notificationDoc,
 	}}
 
+	if newDoc.Status == ActionPending {
```

Comment:
should this be `activeStatus.Contains(newDoc.Status)`
we want to include "running" and "aborting" too right?

Reply:
I thought we only want to notify uniters to process pending actions but not for `running` or `aborting` ones.

Reply:
You are right. We need the notification doc for any action having non final status(for example, running, aborting, etc.).